gazebo_ros/include/gazebo_ros/node.hpp

```diff
@@ -83,9 +87,11 @@ class GAZEBO_ROS_NODE_PUBLIC Node : public rclcpp::Node
    * </plugin>
    * \endcode
    * \param[in] _sdf An SDF element in the style above or containing a <ros> tag in the style above
-   * \return A shared pointer to a new #gazebo_ros::Node
+   * \param[in] _node_name: An optional node_name to overwrite plugin name being used as node name.
+   * \return A shared pointer to a new #gazebo_ros::Node. A nullptr is returned in case multiple
+   * nodes have the same name resulting in a ROS error and gazebo crash.
```

Comment:
nitpick: Correct me if I'm wrong, but this doesn't actually cause Gazebo to crash; it depends on how the plugin handles the nullptr.

```suggestion
   * \return A shared pointer to a new #gazebo_ros::Node.
   * A nullptr is returned and an error message is logged in case multiple nodes have the same name.
```

Reply:
yeah, make sense.

Reply:
From what I observed from testing is that the gazebo doesn't even started in the first place and the overall process exited cleanly.